### DISCUSSION TITLE
Format of SRT StreamID

### DISCUSSION THREAD

**Discussion by VideoFX** [bluenviron/mediamtx#2869] [Questions]
### Question

Thank you, and great work to the author and community for an amazingly functional and useful project. I am honored to be a part of it.

It would be interesting to customize the format of the streamid.

**Comment by aler9** [comment]
Hello, what's the format of streamid you're interested into? is it the one mentioned in #2469? it this is the case, #2469 will be implemented soon, otherwise provide a sample of the format you want to use.

[a comment by an automated account is not shown]
